mod_deploy/controllers.py

```diff
@@ -135,6 +137,12 @@ def deploy():
         with open('build_commit.py', 'w') as f:
             f.write(build_commit)
 
+        # Update runCI
+        run_ci_repo = path.join(app.config['INSTALL_FOLDER'], 'install', 'ci-vm', 'ci-linux', 'ci', 'runCI')
+        run_ci_nfs = path.join(app.config['SAMPLE_REPOSITORY'], 'vm_data', app.config[
+            'KVM_LINUX_NAME'], 'runCI')
```

Comment:
Why is it broken of like this? Why not just the 'runCI' part?

Reply:
Fixed 😃